lib/src/m_login.dart

```diff
@@ -209,37 +201,76 @@ class MLogin {
   /// Does not require previous Login. In case there is no valid login present
   /// in the browser, the user will be prompted to log in again.
   ///
-  /// Requires [idVerificationRedirectUri] to be set!
-  /// Throws exception if not.
-  ///
   /// It is strongly recommended to keep [loggedInMLoginUserId] in sync with the
   /// logged in user of the calling app to ensure that the profile page is shown
   /// for the correct user.
   ///
+  /// [idVerificationRedirectUri] defines where the (external)
+  /// driver license verification service (e.g. IDNow) should redirect to
+  /// after a successful verification step.
+  ///
   /// Returns [true] in case the user finishes the page using the `done` button,
   /// [false] in any other case (e.g., the user pressed the "cancel" button in
   /// the iOS browser, or the back button on Android). This does not infer any
   /// data change or validation and can safely be ignored.
   ///
-  Future<bool> openDriverLicenseVerification({bool ephemeral = false}) {
-    final idRedirectUri = idVerificationRedirectUri;
-    if (idRedirectUri == null) {
-      throw Exception('"idVerificationRedirectUri" MUST be set when calling '
-          'openDriverLicenseVerification!');
-    }
-
+  Future<bool> openDriverLicenseVerification(
+      {bool ephemeral = false, required String idVerificationRedirectUri}) {
```

Comment:
Auch ein breaking change, wobei die Funktion mWn von niemandem genutzt wird. trotzdem im changelog erwähnen.

Reply:
Auch readme Zeile 306 anpassen bitte :)

Reply:
Was ist in Zeile 306? Ich seh da nur ne Leerzeile...

Reply:
```
> __NOTE__: Make sure that you've set `idVerificationRedirectUri` when configuring your `MLogin`
> object!
```

Reply:
Hmm ok das ist bei mir Zeile 277...

Reply:
Ist angepasst!